SU-SWS/stanford_profile
# D8CORE-1307: Full width page only stylesheet.

# READY FOR REVIEW

# Summary
- This is to fix the mobile (xs, sm, med) left/right margins on a full width page. 

# Needed By (Date)
- Thursday

# Urgency
- Medium

# Steps to Test

1. Pull in this branch.
2. Build and clear caches. 
3. Check a text paragraph and a card on the homepage at xs, sm, md mobile widths. The margins should be the standard 20px, 30px and 50px. 
4. Verify a normal page display looks correct still. 

# Affected Projects or Products
- D8CORE

# Associated Issues and/or People
- D8CORE-1307
- @kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
